Kentico.Kontent.Management.Tests/Unit/ManagementClientTests/CollectionTests.cs

```diff
@@ -0,0 +1,165 @@
+﻿using FluentAssertions;
+using Kentico.Kontent.Management.Models.Collections;
+using Kentico.Kontent.Management.Models.Collections.Patch;
+using Kentico.Kontent.Management.Models.Shared;
+using Kentico.Kontent.Management.Tests.Unit.Base;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Kentico.Kontent.Management.Tests.Unit.ManagementClientTests
+{
+    public class CollectionTests : IClassFixture<FileSystemFixture>
+    {
+        private FileSystemFixture _fileSystemFixture;
+
+        public CollectionTests(FileSystemFixture fileSystemFixture)
+        {
+            _fileSystemFixture = fileSystemFixture;
+            _fileSystemFixture.SetSubFolder("Collection");
+        }
+
+        [Fact]
+        public async Task ListCollections_ListsCollections()
+        {
+            var client = _fileSystemFixture.CreateMockClientWithResponse("Collections.json");
+
+            var expectedItems = _fileSystemFixture.GetExpectedResponse<CollectionsModel>("Collections.json");
+
+            var response = await client.ListCollectionsAsync();
+
+            response.Should().BeEquivalentTo(expectedItems);
+        }
+
+        
+        [Fact]
+        public async void ModifyCollection_Remove_ById_RemovesCollection()
+        {
+            var client = _fileSystemFixture.CreateMockClientWithoutResposne();
+
+            var identifier = Reference.ById(Guid.NewGuid());
+
+            var changes = new[] { new CollectionRemovePatchModel
+            {
+                CollectionIdentifier = identifier
+            }};
+
+            Func<Task> deleteCollection = async () => await client.ModifyCollectionAsync(changes);
+
+            await deleteCollection.Should().NotThrowAsync();
+        }
+
+        
+        [Fact]
+        public async void ModifyCollection_Remove_ByCodename_RemovesCollection()
```

Comment:
The original test was testing that exception was thrown right?

Reply:
See comment above, please.

Reply:
same as above - https://github.com/Kentico/kontent-management-sdk-net/pull/114/files?authenticity_token=HY3Mz8bIm%2FpMydkUhbWA%2F7Wu%2BrMM1GnWe0qZozG%2Bogb7bkIS2HZeNPTZNu4KR8AwxPcutLVztw9Vf3pgto8xFQ%3D%3D&file-filters%5B%5D=.cs&hide-deleted-files=true#r761198377